blackbox/docs/interfaces/http.txt

```diff
@@ -287,7 +287,7 @@ To get more insight into what exactly went wrong an additional ``error_trace``
 GET parameter can be specified to return the stack trace::
 
     sh$ curl -sS -H 'Content-Type: application/json' \
-    ... -X POST '127.0.0.1:4200/_sql?pretty&error_trace=True' -d@- <<- EOF
+    ... -X POST '127.0.0.1:4200/_sql?pretty&error_trace=true' -d@- <<- EOF
```

Comment:
Shouldn't be necessary as this is not case sensitive.

Reply:
it is, because now the request would fail because of strict parsing of the GET parameter